src/pages/index.js

```diff
@@ -168,7 +168,6 @@ export default function IndexPage({ pkgVersion }) {
             selectedTab={selectedTab}
             onChange={(filter) => {
               setSelectedTab(filter);
-              setSearchText('');
```

Comment:
@ynotdraw Tabs were broken and removing this line seem to fix them?

Reply:
Doh, yeah that's invalid.  I should've used typescript 🙈 🤦